{"name": "Проверяем, что если авторизоваться под несуществующим пользователем, запрос возвращает ошибку", "status": "broken", "statusDetails": {"message": "NameError: name 'register_new_courier_and_return_login_password' is not defined", "trace": "self = <test_courier_login.TestCourierLogin object at 0x00000134C13B9100>\n\n    def setup_method(self):\n    \n>       self.login_pass = register_new_courier_and_return_login_password()\nE       NameError: name 'register_new_courier_and_return_login_password' is not defined\n\ntests\\test_courier_login.py:46: NameError"}, "start": 1727539564663, "stop": 1727539564663, "uuid": "a361b726-5a26-4e6a-b67a-60ef8e9f2023", "historyId": "7de980d23e9611efd01ffad46e023e78", "testCaseId": "7de980d23e9611efd01ffad46e023e78", "fullName": "tests.test_courier_login.TestCourierLogin#test_authorization_of_a_non_existent_user", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "DESKTOP-AMN9CM3"}, {"name": "thread", "value": "10628-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}